source/documentation/adrs/adr-008.html.md.erb

```diff
@@ -0,0 +1,94 @@
+---
+title: ADR-008 Deployment environments
+last_reviewed_on: 2023-11-27
+review_in: 12 months
+---
+# <%= current_page.data.title %>
+
+### Date: 27-11-2023
+
+## Status
+
+✅ Accepted
+
+## Context
+
+When working across multiple teams, or switching between teams, it can be hard to follow what each deployment environment is for and what practices it keeps. This can lead to data ending up in an improper environment, or tests being run with incorrect assumptions of the state of the environment.
+
+We sometimes need to test, demo or explore functionality between multiple connected services. To do so, we need persistent environments of each service that are connected together. Not all products have this, and those that do can be inconsistent, making it harder to perform these activities.
+
+## Decision
+
+All services must have the following environments:
+
+- Production environment
+- Deployment testing environment
+- Demo environment
+- Local environment
```

Comment:
Worth adding a note here on expectation of path to live traversal and how those are the core environments for any product.

Reply:
Something like this? I've put them in order and highlighted that they make up the path to live.

> All services must have the following essential environments, which should be deployed sequentially as part of the path to live:
> 
> - Local environment
> - Deployment testing environment
> - Production environment
> - Demo environment